lambdas/service/app.py

```diff
@@ -645,46 +643,90 @@ def generate_manifest(event, context):
     return {'Location': response.headers['Location']}
 
 
-@app.route('/dss/files/{uuid}', methods=['GET'], cors=True)
+proxy_endpoint_path = "/fetch/dss/files"
+
+
+@app.route(proxy_endpoint_path + '/{uuid}', methods=['GET'], cors=True)
 def files_proxy(uuid):
     """
-    Proxies requests to DSS' /files endpoint, with the intend of implementing Retry-After on the client's behalf. The
-    file URL that this service returns on the /repository/files endpoint points at this endpoint. This proxy endpoint
-    was added to prevent browsers that don't implement Retry-After from quickly exhausting their limit on the number
-    of retries.
+    Initiate checking out file for download from the data store
 
-    This is only a best effort since we can only wait at most 30s before API Gateway times out.
+    parameters:
+        - name: uuid
+          in: path
+          type: string
+          description: UUID of the file to be checked out
+          - name: filters
+          in: path
+          type: string
+          description: Filters to be applied when requesting the file download
+
+    :return: A 200 response with a JSON body describing the status of the checkout from DSS.
+    The `uuid` parameter and `filters` are passed to the DSS in order to initiate checkout of the correct file
+    in the correct replica. For more information refer to https://dss.data.humancellatlas.org under GET `/files/{uuid}`
```

Comment:
This is the first time `replica` is mentioned. There is no mention that all query parameters are forwarded to the DSS even though we talked at length about this.